Annex60/Fluid/HeatExchangers/Examples/BaseClasses/Heater.mo

```diff
@@ -32,7 +32,8 @@ partial model Heater "Base class for example model for the heater and cooler"
     redeclare package Medium = Medium,
     m_flow_nominal=m_flow_nominal,
     addPowerToMedium=false,
-    dynamicBalance=false) "Fan or pump"
```

Comment:
@rubenbaetens : Better is to only assign `energyDynamics` as the model already uses by default `massDynamics=energyDynamics`.